UPGRADE-4.1.md

```diff
@@ -121,3 +121,9 @@ Workflow
  * Deprecated `SupportStrategyInterface` in favor of `WorkflowSupportStrategyInterface`.
  * Deprecated the class `ClassInstanceSupportStrategy` in favor of the class `InstanceOfSupportStrategy`.
  * Deprecated passing the workflow name as 4th parameter of `Event` constructor in favor of the workflow itself.
+
+HttpKernel
+----------
+
+ * Added the `Symfony\Component\HttpKernel\Controller\ArgumentResolver\TraceableValueResolver`
```

Comment:
There is no upgrade path here 😄

Reply:
Should I remove it in that case? It doesn't cause harm to add it, but it's not really of any value to developers as it's primarily internal.

Reply:
That's what I meant actually. There is no BC break to document here, no upgrade path. We don't mention features in `UPGRADE` files 😄

Reply:
done :smile: